client/styles/components/_overlay.scss

```diff
@@ -40,4 +40,8 @@
 .overlay__close-button {
   @include icon();
   padding: #{3 / $base-font-size}rem 0;
+}
+
+#overlay__anchor{
```

Comment:
for this project i would like to apply styles only by classes (and sometimes tags for base styles), no id's.

Reply:
ok,i'll change that